lighthouse-core/config/default.js

```diff
@@ -121,8 +109,6 @@ module.exports = {
     "dobetterweb/external-anchors-use-rel-noopener",
     "dobetterweb/geolocation-on-start",
     "dobetterweb/link-blocking-first-paint",
-    "dobetterweb/no-console-time",
-    "dobetterweb/no-datenow",
     "dobetterweb/no-document-write",
     "dobetterweb/no-mutation-events",
```

Comment:
im seeing this: 
![image](https://cloud.githubusercontent.com/assets/39191/25865978/b6740fb8-34a9-11e7-94b3-cc72808af83f.png)

related to your change?